### PR TITLE
MF-898 - Add transactions to postgres connect

### PR DESCRIPTION
**Pull request by nwneisen**
Signed-off-by: Nick Neisen <nwneisen@gmail.com>

### What does this do?
Adds transactions to connect code for bulk inserts at the database level.

### Which issue(s) does this PR fix/relate to?
Relates to #898.

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
Single connection tests were modified for bulk connections.

### Did you document any new/modified functionality?
Code comments where changed to reflect bulk inserts.

### CODE REVIEW

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

File: things/mocks/channels.go [hunk 1]
```diff
@@ -160,26 +160,29 @@ func (crm *channelRepositoryMock) Remove(_ context.Context, owner, id string) er
 	return nil
 }
 
-func (crm *channelRepositoryMock) Connect(_ context.Context, owner, chanID, thingID string) error {
-	channel, err := crm.RetrieveByID(context.Background(), owner, chanID)
+func (crm *channelRepositoryMock) Connect(_ context.Context, owner string, chID string, thIDs ...string) error {
```
Comment (manuio) [inline things/mocks/channels.go]: Use shorthand parameters notation please: ` owner, chID string`

**Review by manuio — CHANGES_REQUESTED** [review]

File: things/tracing/channels.go [hunk 2]
```diff
@@ -91,12 +91,12 @@ func (crm channelRepositoryMiddleware) Remove(ctx context.Context, owner, id str
 	return crm.repo.Remove(ctx, owner, id)
 }
 
-func (crm channelRepositoryMiddleware) Connect(ctx context.Context, owner, chanID, thingID string) error {
+func (crm channelRepositoryMiddleware) Connect(ctx context.Context, owner string, chanID string, thingID ...string) error {
```
Comment (manuio) [inline things/tracing/channels.go]: Same here. Use shorthand parameters notation please: `owner, chanID string,`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**